hercules-ci-api/src/Hercules/API/Organizations/Organization.hs

```diff
@@ -0,0 +1,13 @@
+{-# LANGUAGE DeriveAnyClass #-}
+
+module Hercules.API.Organizations.Organization where
+
+import Hercules.API.Prelude
+
+data Organization
+  = Organization
+      { id :: Id Organization,
+        displayName :: Text,
+        trialStartedOn :: Maybe UTCTime
+        }
```

Comment:
List of accounts would be nice but probably not necessary yet.

Reply:
Yes, going to add that once needed - probably next month :)